urdfenvs/urdfCommon/urdf_env.py

```diff
@@ -229,47 +256,94 @@ def add_goal(self, goal) -> None:
-            p.getQuaternionFromEuler([0, 0, 0]),
-        )
-        p.createMultiBody(
-            0,
-            col_wall_id,
-            10,
-            [0, limits[0][1], 0.0],
-            p.getQuaternionFromEuler([0, 0, np.pi / 2]),
-        )
-        p.createMultiBody(
-            0,
-            col_wall_id,
-            10,
-            [0, limits[1][1], 0.0],
-            p.getQuaternionFromEuler([0, 0, np.pi / 2]),
-        )
+        self.add_shapes(shape_type="GEOM_BOX", dim=dim, mass=0, poses_2d=poses_2d)
+
+
+    def add_shapes(self, shape_type: str, dim=None, mass=10, poses_2d=[[-2, 2, 0]], place_height=None) -> None:
+        """
+        Adds a shape to the simulation environment.
+
+        Parameters
+        ----------
+
+        shape_type: shape type, options are "GEOM_SPHERE", "GEOM_BOX", "GEOM_CYLINDER", "GEOM_CAPSULE"
+        dim: dimensions for the shape, dependent on the shape_type:
+            GEOM_SPHERE,    dim=[radius],                   type np.ndarray or list
+            GEOM_BOX,       dim=[width, length, height],    type np.ndarray or list
+            GEOM_CYLINDER,  dim=[radius, length],           type np.ndarray or list
+            GEOM_CAPSULE,   dim=[radius, length],           type np.ndarray or list
+        mass: objects mass
+        poses_2d: list of [[x_position, y_position, orientation)], ...] to place the objects
+        place_height: z_position of the center of mass
+            if place_height = None then the shape will be placed against the ground plane
+        """
+        # convert list to numpy array
+        if isinstance(dim, list):
+            dim = np.array(dim)
+
+        # create collisionShape
+        if shape_type == "GEOM_SPHERE":
+            # check dimensions
+            dim = check_shape_dim(dim, "GEOM_SPHERE", 1, default=np.array([0.5]))
+            shape_id = p.createCollisionShape(p.GEOM_SPHERE, radius=dim[0])
+
+        elif shape_type == "GEOM_BOX":
+            if dim is not None:
+                dim = 0.5 * dim
+            # check dimensions
+            dim = check_shape_dim(dim, "GEOM_BOX", 3, default=np.array([0.5, 0.5, 0.5]))
+            shape_id = p.createCollisionShape(p.GEOM_BOX, halfExtents=dim)
+
+        elif shape_type == "GEOM_CYLINDER":
+            # check dimensions
+            dim = check_shape_dim(dim, "GEOM_CYLINDER", 2, default=np.array([0.5, 1.0]))
+            shape_id = p.createCollisionShape(p.GEOM_CYLINDER, radius=dim[0], height=dim[1])
+
+        elif shape_type == "GEOM_CAPSULE":
+            # check dimensions
+            dim = check_shape_dim(dim, "GEOM_CAPSULE", 2, default=np.array([0.5, 1.0]))
+            shape_id = p.createCollisionShape(p.GEOM_CAPSULE, radius=dim[0], height=dim[1])
+
+        else:
+            warnings.warn("Unknown shape type: {}, aborting...".format(shape_type))
+            return
+
+        # if place_height == None, place against ground plane
+        if place_height is None:
```

Comment:
This is a duplicate if-clause to previously. This should be avoided.

Reply:
_Why didn't you use the add_obstacle function that was already there?_
Add obstacle does require that the obstacle if of class: CollisionObstacle
for which I currently do not see the additional advantages. 

Additionally:
``obst1Dict = {
    "dim": 3,
    "type": "sphere",
    "geometry": {"position": [2.0, 2.0, 1.0], "radius": 1.0},
}
sphereObst1 = SphereObstacle(name="simpleSphere", contentDict=obst1Dict)

env.add_obstacle(sphereObst1)``

is a lot more code then:
`` env.add_shapes(shape_type="GEOM_SPHERE", dim=[1.0], mass=5, poses_2d=[[2.0, 2.0, 1.0]])``

_Than if-clauses can be done entirely in MotionPlanningEnv?_
The real reason is I do not know motionPlanningEnv and what it would provide. Having all objects in one place is better than 2 different places, so we can look at creating every obstacle in the MotionPlanningEnv and having only a add_obstacle function in the  gym_urdf_envs repo.